Hoon/AndroidStudy_Hoon/app/src/main/java/com/study/androidstudy_hoon/SearchActivity.kt

```diff
@@ -0,0 +1,63 @@
+package com.study.androidstudy_hoon
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Menu
+import android.widget.SearchView
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.study.androidstudy_hoon.data.repository.GithubSearchRepoImpl
+import com.study.androidstudy_hoon.data.service.GithubServiceBuilder
+import com.study.androidstudy_hoon.databinding.ActivitySearchBinding
+import com.study.androidstudy_hoon.domain.usecase.SearchRepoUseCase
+import com.study.androidstudy_hoon.presenter.search.SearchAdapter
+import com.study.androidstudy_hoon.presenter.search.SearchViewModel
+import com.study.androidstudy_hoon.presenter.search.SearchViewModelFactory
+
+class SearchActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivitySearchBinding
+    private lateinit var viewModel: SearchViewModel
+    private lateinit var adapter: SearchAdapter
```

Comment:
by lazy로 선언 가능해보여요~
SearchAdapter선언시에 context를 넘기는 이유가 있을까요?

adapter란 네이밍은 불분명하니 searchAdapter로 변경 추천드립니다~

```suggestion
private var searchAdapter by lazy { SearchAdapter() }
```

Reply:
수정완료했습니다 👍